overlay-ports/liblzma/add_support_ios.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
I’d call the new directory smth like “vcpkg-overlay-ports” to make it clearer.

Reply:
That was my original plan, but since "vcpkg*/" is gitignored, I renamed it. I guess I can force it... or should we come with another name?

Reply:
> but since "vcpkg*/" is gitignored

hmpf right! then we can reverse it maybe, like `overlay-ports-vcpkg`, that should go through .gitignore